provisioning/miniprov/hare_mp/main.py

```diff
@@ -37,6 +37,10 @@
 from hare_mp.store import ConfStoreProvider
 from hare_mp.validator import Validator
 
+# Logger details
+log_file = "/var/log/seagate/hare/setup.log"
+log_directory = "/var/log/seagate/hare/"
```

Comment:
- rename `log_directory` to `LOG_DIR`.
- rename `log_file` to `LOG_FILE`
- set LOG_FILE = os.path.join(LOG_DIR, 'setup.log')